readme.md

```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
+If you would like instructor feedback after your project has been completed, please **submit a link to your Github repository as a Git issue to the [`wdi-project4-gallery`](https://github.com/ga-dc/wdi-project4-gallery) repository**. If you do so **by 6:00pm on Friday**, we will do our best to have feedback returned to you some time the following week. If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know.
 
-As always, instructors will be available for 1-1 appointments and support. Please **submit technical issues to the `pbj-project4` repository as Git issues**. To schedule an appointment, please Slack the relevant instructor.
+As always, instructors will be available for 1-1 appointments and support. Please **submit technical issues to the `project4` repository as Git issues**. To schedule an appointment, please Slack the relevant instructor.
 
 ---
 
 ### Presentations
-
 These will be "Science Fair-style" on Friday.
 
 ---
-
-### Dates and times
-
-- Monday
-  - 2:00 - 3:00pm: You and your squad will scrum with your instructor, followed by optional code reviewing with your squadmates. Your squad instructor will take attendance.
-- Wednesday
-  - 2:00 - 3:00pm: You and your squad will scrum with your instructor, followed by optional code reviewing with your squadmates. Your squad instructor will take attendance.
-- Friday
-  - 12:00: Attendance will be taken. You and another student will pair to peer-review each others' code.
-  - 12:30 Lunch is provided!
-  - 3:00: Squads Ada, Bash, and C will present at the Science Fair.
-  - 3:30: Break
-  - 3:45: Squads Dart, Elixir, and Fortran will present at the Science Fair.
-  - 4:15: Break
-  - 4:30: The class will debrief as a whole.
-  - 5:30: The start of the rest of your lives! (And Happy Hour!)
+### Schedule - Dates and times
+#### Fri - 8 Jan
+|time| what's happening|
+|---|---|
+|9:00| Project 4 Kickoff. Pick whichever classroom you want! |
+|9:00 - 11:00| Project Brainstorm - user stories, some initial sketches/wireframing, possible ERD's|
+|11:00| Squad Scrum - Bounce stories off squad and instructor
+|11:00 - 4:00| Work on projects |
+|4:00| Feed BACK(robin's activity)|
+|5:00| Squad Scrum - Discuss project 1 |
+
+#### Mon - 11 Jan
+Schedule 15 minute appt with instructor. Attendance will be taken at appt by squad leader.
+
+#### Tue(12 Jan) - Thur(14 Jan)
+Schedule 30 minute appt with instructor. Attendance will be taken at appt by squad leader.
+
+#### Friday
+|time| what's happening|
+|---|---|
+|9:00| breakfast? |
+|12:00| Pencil's Down! Lunch is provided! |
+|1:00 - 1:45| NaN, AwesomeAnimalHomies, FullSquid`<div>`elopers presents |
+|1:45 - 2:00| Break! |
+|2:00 - 2:45| Undefined, DrJammer, Hackasaurs presents |
+|2:45 - 3:00| Break! |
+|3:00 - 3:45| `<br>`ingBad, LearningToShred, Skwahd presents |
+|4:00 - 4:30| Final Squad Scrum |
+|4:30 - 5:00| End Course Feedback |
+|5:00| The class will debrief as a whole.|
```

Comment:
@bmartinowich Just a heads up -- I remember some butting of heads between UX and Campus Experience over space while UXDI's final retro overlapped with setup for Hacker Hustle Designer, seemly due to lack of pre-crunchtime communication on both side.